theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -2,7 +2,7 @@ void git_clone_foreman_infra(args = [:]) {
     target_dir = args.target_dir ?: ''
```

Comment:
It's odd to name it `git clone foreman-infra` and then you end up with `jenkins-jobs`